### ISSUE TITLE
Pull out `@salesforce` import transforms not relevant to OSS

### ISSUE THREAD

**Issue by trevor-bliss** [salesforce/lwc-test#44]
In a test, the `@salesforce` scoped module imports will be automatically transpiled to provide a default value since the actual value would require a connection to an active app.

For example, `import myMethod from '@salesforce/apex/FooController.fooMethod'` will be automatically transpiled to define `myMethod` and set it's value to some reasonable default. In this case, `myMethod` will be a function that returns a resolved Promise.

Most (all?) of these `@salesforce` imports are not relevant to people not devloping on the Salesforce platform. We should pull these transforms out into a separate project and only apply them when tests are run on the platform.

**Comment by pmdartus** [comment]
Agreed, all the `@salesforce/` prefixed imports should be transformed by another transformer. In order to keep this project simple to maintain, what do you think about creating this transformer in this repository?

**Comment by trevor-bliss** [comment]
I don't mind keeping the additional transformer in this repo, though I know some imports may be internal only.

**Comment by CommanderQ** [comment]
Would I be correct in assuming that resolving this issue would also be related to enhanced support for the `@salesforce/internal/*` imports?
